test/acceptance/nouns.ts

```diff
@@ -0,0 +1,202 @@
+      });
+
+      // 2. Name owner registers nouns.eth as a new guild
+      // const nounsDomain = "nouns.eth";
+      await asAccount(nounsDomainOwner, async (signer) => {
+        // Register guild
+        await ensGuilds
+          .connect(signer)
+          .registerGuild(nounsDomain, admin, flatFeePolicy.getAddress(), openAuthPolicy.getAddress());
+      });
+
+      // 3. Name owner configures NFTWildcardResolver to resolve nouns.eth names using Nouns token contract
+      await asAccount(nounsDomainOwner, async (signer) => {
+        await erc721WildcardResolver
+          .connect(signer)
+          .setTokenContract(nounsDomain, nounsToken.getAddress(), originalResolver);
+      });
+
+      // 4. Guild admin registers NFTWildcardResolver as the fallback resolver for the nouns.eth guild
+      await asAccount(admin, async (signer) => {
+        await ensGuilds.connect(signer).setFallbackResolver(nounsHash, erc721WildcardResolver.getAddress());
+      });
+    });
+
+    describe("Nouns token ID sub-names", function () {
+      it("Are automatically assigned to existing Noun holders", async function () {
+        const { ensRegistry } = this.deployedContracts;
+        const existingNounId = 0;
+        const nounOwner = await nounsToken.ownerOf(existingNounId);
+
+        // Check that 0.nouns.eth resolves to the address of the owner
+        const resolvedAddr = await resolveAddr(ensRegistry, `${existingNounId}.${nounsDomain}`);
+        expect(resolvedAddr).to.eq(nounOwner);
+      });
+      it("Automatically follow transfers of Noun ownership", async function () {
+        const [minter] = await getUnnamedAccounts();
+        const { ensRegistry } = this.deployedContracts;
+        const existingNounId = 0;
+        const nounOwner = await nounsToken.ownerOf(existingNounId);
+        await setBalance(nounOwner, parseEther("100000000"));
+
+        // Transfer the Noun to a new account
+        await asAccount(nounOwner, async (signer) => {
+          await nounsToken.connect(signer).transferFrom(nounOwner, minter, existingNounId);
+        });
+
+        // Check that 0.nouns.eth resolves to the account it was transferred to
+        const resolvedAddr = await resolveAddr(ensRegistry, `${existingNounId}.${nounsDomain}`);
+        expect(resolvedAddr).to.eq(minter);
+      });
+      it("Are automatically assigned on auctions of new Nouns", async function () {
+        const { ensRegistry } = this.deployedContracts;
+        const [minter] = await getUnnamedAccounts();
+        const nounsAuctionOwner = await nounsAuction.owner();
+
+        // bid on current auction
+        const nounId = (await nounsAuction.auction()).nounId;
+        await asAccount(minter, async (signer) => {
+          await nounsAuction.connect(signer).createBid(nounId, { value: ethers.parseUnits("100", "ether") });
+        });
+
+        // set date to after auction has closed
+        const date = new Date();
+        date.setDate(date.getDate() + 1);
```

Comment:
Curious why we're using the current date instead of the blockchain fork's "current" date, and why we're adding 1 to the current date